src/main/java/edu/cornell/kfs/paymentworks/PaymentWorksConstants.java

```diff
@@ -18,28 +18,60 @@
  */
 package edu.cornell.kfs.paymentworks;
 
+import org.apache.commons.lang.StringUtils;
+
+import edu.cornell.kfs.fp.CuFPConstants.ScheduledSourceAccountingLineConstants.ScheduleTypes;
+
 public class PaymentWorksConstants {
+	
+	public static final String PAYMENT_WORKS_FIELD_MAPPING_DOCUMENT_TYPE = "PMWM";
 
-	public static final String PAYMENT_WORKS_VENDOR_SEQ = "PAYMENT_WORKS_VENDOR_SEQ";
+	public static final String PAYMENT_WORKS_VENDOR_SQ = "PAYMENT_WORKS_VENDOR_SQ";
+	public static final String PAYMENT_WORKS_FIELD_MAPPING_SQ = "PAYMENT_WORKS_FIELD_MAPPING_SQ";
```

Comment:
SEQ is consistent with our database naming conventions, but after looking at other code, it looks like _SEQUENCE_NAME is used for constants (although not completely consistently). So I would propose PAYMENT_WORKS_VENDOR_SEQUENCE_NAME and PAYMENT_WORKS_FIELD_MAPPING_SEQUENCE_NAME for these.